src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1382,6 +1370,105 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Partial Class BoundFieldOrPropertyInitializer
+        Implements IFieldInitializer, IPropertyInitializer
```

Comment:
I think implementing two interfaces on this node could be confusing to consumers. I am not sure why do we expose this node at all since we added BoundPropertyEqualsValue and BoundFieldEqualsValue.